.github/workflows/build-package.yml

```diff
@@ -0,0 +1,60 @@
+name: Build Package
+
+env:
+  MAIN_BRANCH: "all-citus"
+  PACKAGE_CLOUD_REPO_NAME: "sample"
+
+on:
+  push:
+    branches: "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ el/7,
+                    el/8,
+                    ol/7,
+                    debian/stretch,
+                    debian/buster,
+                    ubuntu/xenial,
+                    ubuntu/bionic,
+                    ubuntu/focal ]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Clone tools branch
+        run: git clone --depth=1 --branch develop https://github.com/citusdata/tools.git tools
```

Comment:
I suggest you bump a new version on tools repo and use that tag so that it will not break as easily in the future.

Reply:
Fixed